File: IOS/SAIOT/SAIOT/DataModel/RemoteList.swift
//
//  KitxhenSensor.swift
//  SAIOT
//
//  Created by 양창엽 on 2018. 4. 2..
//  Copyright © 2018년 Yang-Chang-Yeop. All rights reserved.
//

import RealmSwift

class RemoteList: Object {
    
    // MARK: - Date Value
    @objc dynamic var usedDate: Date = Date() 
    
    // MARK: - Integer Value
    @objc dynamic var hue: Int = 0
    @objc dynamic var colors: [Int] = [0, 0, 0]
    
    // MARK: - String Value
    @objc dynamic var hueBridgeID: String = ""
    @objc dynamic var hueBridgeIP: String = ""
    @objc dynamic var hueBridgeName: String = ""
    
    // MARK: - List
    let controllerList = List<RemoteControllerInfo>()
}

File: Android/SAIOT/app/src/main/java/androidtown/org/saiot/RemoteList.java
package androidtown.org.saiot;

import java.util.Date;

import io.realm.RealmObject;

/**
 * Created by Ei Seok on 2018-04-24.
 */

public class RemoteList extends RealmObject {

    //Date Value
    private Date usedDate= new Date();

    //Integer Value
    private int hue;
    private int colors;

    //String Value
    private String hueBridgeID;
    private String hueBridgeIP;
    private String hueBridgeName;

    //List

    //let controllerList = List<RemoteControllerInfo>()





}
